sunbird-es-utils/src/main/resources/indices/cbatch.json

```diff
@@ -331,6 +331,14 @@
                         }
                     }
                 },
+                "enrollmentEndDate": {
```

Comment:
Do not include new fields in the index creation json, it should all go in mapping file only